src/main/java/rx/internal/operators/ThrowObservable.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package rx.internal.operators;
+
+import rx.*;
+
+/**
+ * An Observable that invokes {@link Observer#onError onError} when the {@link Observer} subscribes to it.
+ * 
+ * @param <T>
+ *            the type of item (ostensibly) emitted by the Observable
+ */
+public final class ThrowObservable<T> extends Observable<T> {
```

Comment:
Is this type needed? Why not just the OnSubscribe class which is passed to create()?